test/common/json/json_loader_test.cc

```diff
@@ -78,7 +78,8 @@ TEST_F(JsonLoaderTest, Basic) {
     EXPECT_THROW(json->getBoolean("hello"), Exception);
     EXPECT_THROW(json->getObjectArray("hello"), Exception);
     EXPECT_THROW(json->getString("hello"), Exception);
-
+    EXPECT_THROW_WITH_MESSAGE(json->getString("hello",""), Exception,
```

Comment:
Added test case for getString with default. Appears to be no users outside of contrib any more.